### ISSUE TITLE
Game crashes in character selection menu

### ISSUE THREAD

**Issue by ZhilkinSerg** [CleverRaven/Cataclysm-DDA#21974]
Game crashes when moving cursor in `Skills` tab in character creation menu to `rifle` skill (when moving cursor downward) or `unarmed` skill (when moving cursor upward).

Experimental 6793 on Windows Tiles without mods.

The issue appears with terminal size 120 x 40 and 80 x 40.
The issue does not appear with terminal size 80 x 25.

Probably appeared after #21835.

**Comment by ZhilkinSerg** [comment]
Probably a duplicate of #21960.
